Homework_4/seminar_task_8.py

```diff
@@ -0,0 +1,24 @@
+"""✔ Создайте несколько переменных заканчивающихся и не оканчивающихся на «s».
+✔ Напишите функцию, которая при запуске заменяет содержимое переменных
+оканчивающихся на s (кроме переменной из одной буквы s) на None.
+✔ Значения не удаляются, а помещаются в одноимённые переменные без s на конце.
+"""
+
+
+def func():
+    dct = globals()
+    for key in [*dct.keys()]:
+        if key.endswith('s') and len(key) > 1:
+            dct[key.rstrip('s')], dct[key] = dct[key], None
```

Comment:
Еще можно просто срез использовать.
```suggestion
            dct[key[:-1]], dct[key] = dct[key], None
```